woocommerce/woocommerce-ios
# Order List: Filters

Task #30 

**To Test**
- navigate to Orders
- tap on filter button in navigation 
- select any filter button (except "All")
- if there are orders with that payment status, they should display
- if there are no orders with that payment status, a "no results" message displays
- selecting "All" after filtering orders will display all the orders again

**Bonus Bug Fix to Test**
- navigate to Orders
- pull up on table to reveal search bar
- tap inside search bar so it is active
- instead of typing a search into the search bar, tap on an order
<img src="https://user-images.githubusercontent.com/1062444/40189947-3b934102-59c3-11e8-909b-124e490ef5d3.png" width="250" />


**Expected Behavior**
the search bar dismisses and the screen navigates to the order detail
<img src="https://user-images.githubusercontent.com/1062444/40189929-326b5fec-59c3-11e8-946f-5539fb77f6e3.png" width="250" />

**Actual Behavior**
the screen was navigating to the order detail and leaving the search bar layered on top 
<img src="https://user-images.githubusercontent.com/1062444/40189924-2e828cca-59c3-11e8-99a5-8b9cc7ab99e6.png" width="250" />
